### ISSUE TITLE
NVDA is not able to read the text of list items in Total Commander's 'Change Start Menu' dialogue

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#1724]
**Reported by oaron on 2011-08-17 10:40**
In Total Commander's 'Change Start Menu' dialogue the entries are presented in a list.
NVDA is not able to read the text of the list entries, however it's able to retrieve them (e.g. via nvda+ctrl+f2)
TC can be downloaded from [here.](http://www.ghisler.com/)

**Comment by bhavyashah** [comment]
Since the latest stable release of Total Commander for Windows 32+64 bit was about 7 months ago, I would consider this software's development status active. However, we cannot contact the original author of this ticket, the reported issue is almost six years old now, and this may have been resolved in recent versions of both TC and NVDA in this last six years. Time to close? @ehollig

**Comment by k-kolev1985** [comment]
Actually, this seams to be fixed. I can now read the list items in the "Change start menu" dialog of Total Commander. The test is done on a 64-bit OS. I have both the 32 and the 64-bit versions of Total Commander v9.0a installed. In the 64-bit edition of Total Commander everything works as expected. In the 32-bit edition there is a minor annoyance - NVDA says "selected" in front of every entry in the "Start menu" list, but otherwise it works as expected.

**Comment by ehollig** [comment]
From https://github.com/nvaccess/nvda/issues/1724#issuecomment-320680561, closing as worksforme
